src/types/redis_hash.cc

```diff
@@ -58,7 +65,10 @@ rocksdb::Status Hash::Get(const Slice &user_key, const Slice &field, std::string
   rocksdb::ReadOptions read_options;
   read_options.snapshot = ss.GetSnapShot();
   std::string sub_key = InternalKey(ns_key, field, metadata.version, storage_->IsSlotIdEncoded()).Encode();
-  return storage_->Get(read_options, sub_key, value);
+  s = storage_->Get(read_options, sub_key, value);
+  if (!s.ok()) return s;
+  uint64_t expire = 0;
+  return decodeFieldValue(metadata, value, expire);
```

Comment:
Is it OK, that `decodeFieldValue` doesn't assign something to `value` if the data structure doesn't have the `METADATA_HASH_FIELD_EXPIRE_MASK` flag?

Reply:
Yes. Inside the `decodeFieldValue`, if doesn't have `METADATA_HASH_FIELD_EXPIRE_MASK` flag will return directly and don't do anything.